UIKitDZ/UIKitDZ/PostScreen/PostsScreen/PostsViewController.swift

```diff
@@ -0,0 +1,203 @@
+// PostsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Класс для отображения главноего экрана с поставми
+class PostsViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let identifierStories = "StoriesCell"
+        static let identifierPosts = "PostCell"
+        static let identifierRecomendations = "RecomindationCell"
```

Comment:
сущность всегда стоит последним словом - напр. PostsViewController это ViewController, SubscribeCell это Cell
```suggestion
        static let storiesIdentifier = "StoriesCell"
        static let postsIdentifier = "PostCell"
        static let recommendationsIdentifier = "RecomindationCell"
```